### ISSUE TITLE
Errors in HA log

### ISSUE THREAD

**Issue by mipe79** [jockesyk/homeassistant-yale-doorman-via-smarthub#16]
I get the following errors when I start HA with this component. I don't know if it affect any functionality.

```
Error adding entities for domain lock with platform yale_doorman_via_smarthub
Error while setting up yale_doorman_via_smarthub platform for lock
Traceback (most recent call last):
  File "/config/custom_components/yale_doorman_via_smarthub/lock.py", line 97, in icon
    door_status = int(self.coordinator.data["data"]["device_status"][self.idx]["minigw_lock_status"],16)
ValueError: invalid literal for int() with base 16: ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 322, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 434, in _async_add_entity
    original_icon=entity.icon,
  File "/config/custom_components/yale_doorman_via_smarthub/lock.py", line 100, in icon
    door_status = self.old_door_status
AttributeError: 'YaleDoormanViaSmarthubLock' object has no attribute 'old_door_status'
```
```
Error adding entities for domain binary_sensor with platform yale_doorman_via_smarthub
Error while setting up yale_doorman_via_smarthub platform for binary_sensor
Traceback (most recent call last):
  File "/config/custom_components/yale_doorman_via_smarthub/binary_sensor.py", line 60, in icon
    door_status = int(self.coordinator.data["data"]["device_status"][self.idx]["minigw_lock_status"],16)
ValueError: invalid literal for int() with base 16: ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 322, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 434, in _async_add_entity
    original_icon=entity.icon,
  File "/config/custom_components/yale_doorman_via_smarthub/binary_sensor.py", line 63, in icon
    door_status = self.old_door_status
AttributeError: 'YaleDoormanViaSmarthubBinarySensor' object has no attribute 'old_door_status'
```

**Comment by mrbardman** [comment]
Same here. Getting the "ValueError: invalid literal for int() with base 16:" on every server startup. 

The door lock seems to work normally anyway.

**Comment by jockesyk** [comment]
This issue is fixed in version 0.0.8 #30
